stdlib/sysconfig.pyi

```diff
@@ -14,12 +16,21 @@ __all__ = [
     "parse_config_h",
 ]
 
+if sys.version_info >= (3, 10):
+    __all__ += ["get_default_scheme", "get_preferred_scheme"]
+
 def get_config_var(name: str) -> str | None: ...
 @overload
 def get_config_vars() -> dict[str, Any]: ...
 @overload
 def get_config_vars(arg: str, *args: str) -> list[Any]: ...
 def get_scheme_names() -> tuple[str, ...]: ...
+
+if sys.version_info >= (3, 10):
+    def get_default_scheme() -> str: ...
+    _Key: TypeAlias = Literal["prefix", "home", "user"]
+    def get_preferred_scheme(key: _Key) -> str: ...
```

Comment:
Not sure I really see the value of using a type alias here, given that it's only used once

```suggestion
    def get_preferred_scheme(key: Literal["prefix", "home", "user"]) -> str: ...
```